src/redux/reducers/ConceptReducers.js

```diff
@@ -89,6 +89,8 @@ export default (state = initialState, action) => {
         ...state,
         paginatedConcepts: state.paginatedConcepts
           .filter(concept => concept.version_url !== action.payload),
+        dictionaryConcepts: state.dictionaryConcepts
```

Comment:
Is this part of the ticket?

Reply:
Yes, it is. In the store `dictionaryConcepts` wasn't updating after deleting a concept and this was the cause of the issue in the ticket.